create/create.go

```diff
@@ -27,7 +27,7 @@ type Cmd struct {
 	Project           projectCmd           `cmd:"" group:"management.nine.ch" name:"project" help:"Create a new project."`
 	Config            configCmd            `cmd:"" group:"deplo.io" name:"config"  help:"Create a new deplo.io Project Configuration. (Beta - requires access)"`
 	Application       applicationCmd       `cmd:"" group:"deplo.io" name:"application" aliases:"app" help:"Create a new deplo.io Application. (Beta - requires access)"`
-	Redis             redisCmd             `cmd:"" group:"storage.nine.ch" name:"redis" help:"Create a new Redis instance."`
+	KeyValueStore     keyValueStoreCmd     `cmd:"" group:"storage.nine.ch" name:"keyvaluestore" help:"Create a new KeyValueStore instance"`
```

Comment:
Maybe add aliases for `kv`? Also for `redis`?
https://github.com/ninech/nctl/blob/92f24ec26e9a823db93162ad2736b51767dbb170/get/get.go#L22

Reply:
Added `kvs` alias for all CRUD operations.